File: tests/fixtures/python/boolean_operators.py
# Expected metrics:
# check_all: cognitive=TBD, cyclomatic=TBD, sloc=TBD
def check_all(a, b, c):
    if a and b and c:
        return True
    return False

# check_mixed: cognitive=TBD, cyclomatic=TBD, sloc=TBD
def check_mixed(a, b, c):
    if a and b or c:
        return True
    return False

File: tests/fixtures/go/boolean_operators.go
package fixtures

// Expected metrics:
// checkAll: cognitive=TBD, cyclomatic=TBD, sloc=TBD
func checkAll(a bool, b bool, c bool) bool {
	if a && b && c {
		return true
	}
	return false
}

// checkMixed: cognitive=TBD, cyclomatic=TBD, sloc=TBD
func checkMixed(a bool, b bool, c bool) bool {
	if a && b || c {
		return true
	}
	return false
}
